common.mk

```diff
@@ -17,7 +17,7 @@ CHOWN:=docker run --rm -v $(CURDIR):/v -w /v alpine chown
 DEFAULT_PRODUCT_LICENSE:=Community Engine
 PACKAGER_NAME?=
 DOCKER_GITCOMMIT:=abcdefg
-GO_VERSION:=1.20.8
+GO_VERSION:=1.21.0
```

Comment:
Can we split this into a separate commit (so "bump Go" then "bump Compose")?

Reply:
Sure

Reply:
reverted